Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:jessie
+
+ENV ARCH=x86_64-unknown-linux-gnu
+ENV RUST_RELEASE=1.7.0
+ENV CARGO_RELEASE=nightly
+
+RUN echo "deb http://llvm.org/apt/jessie/ llvm-toolchain-jessie main" > /etc/apt/sources.list.d/llvm.list && \
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 15CF4D18AF4F7421 && \
+    apt-get update && \
+    apt-get install -y curl llvm-3.8 vim gcc libssl-dev libedit-dev && \
```

Comment:
In case of alpine, [only llvm 3.8 is bringed ATM](https://pkgs.alpinelinux.org/packages?name=%25llvm%25&repo=all&arch=x86_64&maintainer=all)